lighthouse-cli/scripts/run-smoke-tests.sh

```diff
@@ -2,11 +2,9 @@
 
 cd lighthouse-cli/test/fixtures && python -m SimpleHTTPServer 9999 &
 
-if [[ $(node -v) =~ ^v4.* ]]; then
-  node --harmony lighthouse-cli/index.js http://localhost:9999/online-only.html > results
-else
-  ./lighthouse-cli/index.js http://localhost:9999/online-only.html > results
-fi
+NODE=$([ $(node -v | grep -E "v4") ] && echo "node --harmony" || echo "node")
+
+$NODE ./lighthouse-cli/index.js http://localhost:9999/online-only.html > results
```

Comment:
Do we need the ./ in front of the file path? Because if so shouldn't the second call to it also have it.